install.bash

```diff
@@ -57,26 +58,39 @@ if [ ${SCRATCH_MONGODB_INSTALL} = true ]; then
     if [ -x bin/mongod ]; then
         VERSION_FULL=$(bin/mongod --version | grep 'db version')
         VERSION="${VERSION_FULL:12}"
-        if [ ${VERSION} != "5.0.4" ]; then
-            echo "Upgrading MongoDB v${VERSION} to v5.0.4"
+        if [ "${OS}" == "Darwin" ] && [ "${ARCH}" == "arm64" ]; then
+            if [ ${VERSION} != "6.0.2" ]; then
+                echo "Upgrading MongoDB v${VERSION} to v6.0.2"
+            else
+                echo "MongoDB v6.0.2 already installed"
+                INSTALL_MONGODB=false
+            fi
         else
-            echo "MongoDB v5.0.4 already installed"
-            INSTALL_MONGODB=false
+            if [ ${VERSION} != "5.0.4" ]; then
+                echo "Upgrading MongoDB v${VERSION} to v5.0.4"
+            else
+                echo "MongoDB v5.0.4 already installed"
+                INSTALL_MONGODB=false
+            fi
         fi
     else
         echo "Installing MongoDB v5.0.4"
     fi
     if [ ${INSTALL_MONGODB} = true ]; then
-        if [ "${OS}" == "Darwin" ]; then
-            MONGODB_BUILD=mongodb-macos-x86_64-5.0.4
+        MONGODB_VERSION=5.0.4
+            if [ "${OS}" == "Darwin" ]; then
```

Comment:
Looks like there's some incorrect indentation here?

Reply:
fixed and rebuilt/retested